### ISSUE TITLE
[meta] Rename default branch to main

### ISSUE THREAD

**Issue by nschonni** [nodejs/nodejs.org#3761]
Issue tracking the renames across the org https://github.com/nodejs/node/issues/33864

- [x] PR to change the instances where it is hardcoded https://github.com/nodejs/nodejs.org/pull/3762
- [x] Any changes needed to the nodejs/build webhook setup https://github.com/nodejs/build/pull/2676
- [x] Update Crowdin config?

**Comment by mhdawson** [comment]
Assume this is still underway?

**Comment by nschonni** [comment]
@mhdawson I just sent a PR for the nodejs/build webhook, so I think it's mainly the rename that needs to happen before the 2 PRs are landed.
The Crowdin part isn't really used right now, so it can probably get cleaned up later

**Comment by mhdawson** [comment]
@nschonni I can do the rename if you think everything is ready, just let me know.

**Comment by nschonni** [comment]
I think it should be good to go, as long as there isn't a pending release that might need release notes to get published today. I'm guessing there might be some manual work to redeploy the webhook on the build PR, so the related PR here is picked up

**Comment by mhdawson** [comment]
@BethGriggs what would be a good time to do the rename from the perspective of avoiding releases?

**Comment by BethGriggs** [comment]
It looks like our next set of releases are not scheduled until 29th June, so anytime this week should be a good. But, generally, we aim for Tuesday releases, so just avoiding those seems reasonable.

Edit: Looks like we delayed v16.4.0 until 22nd June  - so worth waiting until that has gone out

**Comment by richardlau** [comment]
We still haven't released https://github.com/nodejs/node/pull/39031 yet (when discussed in last weeks' Release WG meeting the plan was to delay/try for tomorrow (22 June)).

**Comment by mhdawson** [comment]
At this point I think we should likely wait until after the upcoming security release.

**Comment by mhdawson** [comment]
@richardlau do you know what we need to do/have access to do the "might be some manual work to redeploy the webhook on the build PR,"

**Comment by richardlau** [comment]
@mhdawson I'll find out -- I roughly know the bit on the infra servers that rebuild the website but would need to dig through to work out how the webhook works.

**Comment by richardlau** [comment]
@mhdawson For the webhook we'd need to update two files on the www server and then restart the `github-webhook.service` systemd service.

- Update these references to `origin/master`:
  - `ansible/www-standalone/resources/config/github-webhook.json.j2` (https://github.com/nodejs/build/pull/2676)
  - `ansible/www-standalone/resources/scripts/build-site.sh` (requested to be added to https://github.com/nodejs/build/pull/2676)
 - Update the files on `www`. I've never actually run the playbook in https://github.com/nodejs/build/tree/master/ansible/www-standalone before -- the README says it's not tested against the live server and I think we've generally made changes on the server and reflected them back into the Ansible scripts rather than doing them first in Ansible. The two referenced files above end up on the server as:
  - `/etc/github-webhook.json` (this looks different to the one in the repo -- at some point we dropped the webhook for benchmarking but don't appear to have updated the repo)
  - `/home/nodejs/build-site.sh`
- Restart the `github-webhook.service` (e.g. `systemctl restart github-webhook.service`) to reload the configuration.

**Comment by richardlau** [comment]
@mhdawson Also I believe I have access to do the build-infra side of this.

**Comment by mhdawson** [comment]
In respect to 
> Update the files on www. I've never actually run the playbook in https://github.com/nodejs/build/tree/master/ansible/www-standalone before -- the README says it's not tested against the live server and I think we've generally made changes on the server and reflected them back into the Ansible scripts rather than doing them first in Ansible. The two referenced files above end up on the server as:

+1 I think we should do the manual changes and reflect back.

**Comment by mhdawson** [comment]
@richardlau thanks.  Once the PR is updated as you requested, we should find a good time to do the update. Maybe you can look at the upcoming schedule and then confirm with the releasers what a good time would be?

**Comment by mhdawson** [comment]
As a separate thought (not part of this issue/work) it might be good to audit the differences between the files in the repo and what is on the www server. I expect there may be a bunch of differences.

**Comment by nschonni** [comment]
Not to go too far off topic, but I also opened up https://github.com/nodejs/nodejs.org/issues/3018 awhile ago, since we do occasionally have issues with the webhook. But that is separate from this issue. I'll take a look at the build PR comments now

**Comment by richardlau** [comment]
> @richardlau thanks. Once the PR is updated as you requested, we should find a good time to do the update. Maybe you can look at the upcoming schedule and then confirm with the releasers what a good time would be?

The build PR has been updated and looks good. As far as releases go the next planned ones are next Tuesday (21st Sept, 16.x cc @BethGriggs) with a 14 semver minor the week after that (https://github.com/nodejs/node/pull/39990). We could aim to do this tomorrow?

**Comment by mhdawson** [comment]
@richardlau I guess we did not do it today since I see no discussion here.  Would you be able to do it next Wednesday the day after the release?

**Comment by richardlau** [comment]
@mhdawson Yes but I'd need you or another repository owner to do the actual rename here (I can do the build-infra bit).

**Comment by mhdawson** [comment]
I can do the rename, would doing it together at 12 ET work?

**Comment by richardlau** [comment]
> 
> 
> I can do the rename, would doing it together at 12 ET work?

SGTM

**Comment by mhdawson** [comment]
Branch renamed, and we landed a PR (https://github.com/nodejs/nodejs.org/pull/4122) and from what we can see so far we think the webhook ran ok.

**Comment by nschonni** [comment]
Rebased and landed https://github.com/nodejs/nodejs.org/pull/3762 so the "Edit on GitHub" links will work in the footer.
Only outstanding piece seems to be Crowd-in, but I'm unsure how much that ever got used to be honest

**Comment by XhmikosR** [comment]
Can we close this issue now that the branch renaming is done?

As for Crowdin, if it's not used, is there any reason we keep the README.md badge, its config and a PR open?

**Comment by nschonni** [comment]
I believe the Crowdin part was never fixed, but I don't think it's really ever got traction @nodejs/crowdin-managers

**Comment by AugustinMauroy** [comment]
> I believe the Crowdin part was never fixed, but I don't think it's really ever got traction

look up here #4994
